rewrite-java-test/src/test/java/org/openrewrite/java/OrderImportsTest.java

```diff
@@ -709,4 +709,52 @@ public class Foo {
           )
         );
     }
+
+    /**
+     * This style should only reorder the imports but it indents the `extends` keyword as well.
+     * Context is: I'm trying to enforce a custom order of imports, so I don't want this style to do anything else.
+     */
+    @Issue("https://github.com/openrewrite/rewrite/issues/4165")
+    @Test
+    void shouldNotIndentExtends() {
+        rewriteRun(
+          spec -> spec.parser(JavaParser.fromJavaVersion().styles(
+            singletonList(
+              new NamedStyles(
+                randomId(),
+                "it.does.more.than.JustOrderImports",
+                "custom style",
+                null,
+                emptySet(),
+                singletonList(
```

Comment:
Hey @timo-a, I think in your case you also want to influence the tabs and indents like we do [here](https://github.com/openrewrite/rewrite/blob/v8.8.4/rewrite-java/src/main/java/org/openrewrite/java/style/IntelliJ.java#L42) in our IntelliJ style.
Specifically the 4th parameter of the `TabsAndIndentsStyle` [here](https://github.com/openrewrite/rewrite/blob/v8.8.4/rewrite-java/src/main/java/org/openrewrite/java/style/IntelliJ.java#L71) influences the indent spacing